application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/views/FlameGraphView.java

```diff
@@ -282,9 +281,9 @@ public void run() {
 
 	private static class ModelRebuildCallable implements Callable<Void> {
```

Comment:
Does this really have to be callable? Would clean up the code if not.

Reply:
correct, removed callable -> runnable, more executor threads and additional cleanup